src/test/descriptor_tests.cpp

```diff
@@ -28,6 +31,8 @@ constexpr int HARDENED = 2; // Derivation needs access to private keys
 constexpr int UNSOLVABLE = 4; // This descriptor is not expected to be solvable
 constexpr int SIGNABLE = 8; // We can sign with this descriptor (this is not true when actual BIP32 derivation is used, as that's not integrated in our signing code)
 
+
```

Comment:
Unnecessary whitespace change.

Reply:
Thanks, fixed in commit d3e2d08a3be92cecb50c89c69871e61e0618e10b.